pingdom/api_responses.go

```diff
@@ -224,6 +226,12 @@ type CheckResponseTCPDetails struct {
 	StringToExpect string `json:"stringtoexpect,omitempty"`
 }
 
+// CheckResponseDNSDetails represents the details specific to DNS checks.
+type CheckResponseDNSDetails struct {
+	ExpectedIp string `json:"expectedip,omitempty"`
```

Comment:
Would you mind making the variable `ExpectedIP` to match how we capitalize `IP` in the other types?